zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/message/RequestData.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.state.message;
+
+import io.camunda.zeebe.db.DbValue;
+import io.camunda.zeebe.msgpack.UnpackedObject;
+import io.camunda.zeebe.msgpack.property.IntegerProperty;
+import io.camunda.zeebe.msgpack.property.LongProperty;
+
+public class RequestData extends UnpackedObject implements DbValue {
```

Comment:
🔧 Is the name too generic ? Should we specify that this is related to messages ?

Reply:
I don't think that's necessary. The object only stores request data, if we ever need this for a different feature there's no problem with this being reused.

Reply:
Ok, I see. Then should we move this class into another package ?

Reply:
> Ok, I see. Then should we move this class into another package ?

If you have a good suggestion I'm happy to move it. I figured we could move it if we ever actually use it in multiple places.

Reply:
Not really. I'd suggest to leave the class there and if needed we can move it as you said 👍